ush/nems.configure.cpld_aero.IN

```diff
@@ -0,0 +1,128 @@
+#############################################
+####  NEMS Run-Time Configuration File  #####
+#############################################
+
+# ESMF #
+logKindFlag:           @[esmf_logkind]
+globalResourceControl: true
+
+# EARTH #
+EARTH_component_list: MED ATM CHM OCN ICE
+EARTH_attributes::
+  Verbosity = 0
+::
+
+# MED #
+MED_model:                      @[med_model]
+MED_petlist_bounds:             @[med_petlist_bounds]
+MED_omp_num_threads:            @[med_omp_num_threads]
+::
+
+# ATM #
+ATM_model:                      @[atm_model]
+ATM_petlist_bounds:             @[atm_petlist_bounds]
+ATM_omp_num_threads:            @[atm_omp_num_threads]
+ATM_attributes::
+  Verbosity = 0
+  DumpFields = @[DumpFields]
```

Comment:
Do we need this variable in GW? This might only be used for debug purpose. I think it can be set to false.

Reply:
@junwang-noaa I think we are just being consistent w/ the other `nems.configure` templates.  It is default set to `.false.` I think.  If not, we can address in a later PR.